.nuget/directxmesh-config.cmake.in

```diff
@@ -0,0 +1,11 @@
+@PACKAGE_INIT@
+
+if ((NOT directxmesh_FIND_COMPONENTS) OR ("library" IN_LIST directxmesh_FIND_COMPONENTS))
```

Comment:
There's not a lot of guidance on using "Components" for CMake packages, and it seems to be left as an 'exercise to the reader' like many "advanced CMake" scenarios.

Here I decided to use it so that by default (not providing any components), you get the DirectXMesh.lib only which is the 99% use case.

If you want to get to "WaveFrontReader.h", you can get it by adding "library utils".

If you want "WaveFrontReader.h" but don't want DirectXMesh, you add just "utils".

If anyone who knows cmake well thinks this is an abomination, let me know :)